index.html

```diff
@@ -0,0 +1,264 @@
+    <section class="features">
+        <div class="container">
+            <h2 class="features__title">Features</h2>
+
+            <!-- features cards -->
+            <div class="features__cards">
+                <div class="card">
+                    <div class="card__img">
+                        <img class="card__img--link" src="./images/responsive.svg" alt="phone icon">
+                    </div>
+                    <h3 class="card__title">fully responsive</h3>
+                    <p class="card__para">Looks amazing on any device: smartphone, tablet,
+                        laptop and
+                        desktop.</p>
+                </div>
+
+                <div class="card">
+                    <div class="card__img"><img class="card__img--link" src="./images/customizable.svg"
+                            alt="tools icon"></div>
+                    <h3 class="card__title">customizable</h3>
+                    <p class="card__para">Change the colors, pictures or any of the sections
+                        to suit
+                        your needs.</p>
+                </div>
+
+                <div class="card">
+                    <div class="card__img"><img class="card__img--link" src="./images/design.svg" alt="medal icon">
+                    </div>
+                    <h3 class="card__title">slick and beautiful design</h3>
+                    <p class="card__para">Trendy and fresh design, fits any website.</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- *-+*-+*-+*-+*-+*-+*-+*-+*-+ Logos Section *-+*-+*-+*-+*-+*-+*-+*-+ -->
+    <section class="logos">
+        <img src="./images/logos.png" alt="companies logos" class="logos__img">
+    </section>
+
+    <!-- *-+*-+*-+*-+*-+*-+*-+*-+*-+ About Section *-+*-+*-+*-+*-+*-+*-+*-+ -->
+    <section class="about">
+        <div class="container">
+            <div class="about__img">
+                <img class="about__img--link" src="./images/imac.png" alt="computer screen">
+            </div>
+
+            <div class="about__content">
+                <h2 class="about__title">simple and beautiful</h2>
+                <p class="about__para">Use Nova theme for your next web project. It is completely
+                    customizable so you can change any of the sections to fit your needs. Nova Theme is Free for any
+                    kind of use, personal and commercial. Have fun and good luck!</p>
+            </div>
+
+        </div>
+    </section>
+
+    <!-- *-+*-+*-+*-+*-+*-+*-+*-+*-+ Pricing Section *-+*-+*-+*-+*-+*-+*-+*-+ -->
+    <section class="pricing">
+        <div class="container">
+            <h2 class="pricing__title">choose your pricing plan</h2>
+            <p class="pricing__para">Pick any of our super affordable pricing plans</p>
+
+            <!-- pricing cards -->
+            <div class="pricing___content">
+                <div class="pricing__card">
```

Comment:
Ponieważ to lista pakietów cenowych to również można zrobić `ul > li` i zagnieżdżenie pozostałych elementów nie jest problemem :)